imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -154,12 +154,12 @@ class CaseExplorer extends Component {
           sortBy={sortBy}
           roles={['All', 'Assigned to me']}
           labels={[
-            [SORT_BY.DATE_DESCENDING, 'Newest (Created) ↓'],
-            [SORT_BY.DATE_ASCENDING, 'Oldest (Created) ↑'],
-            [SORT_BY.NAME_ASCENDING, 'Name (A to Z) ↑'],
-            [SORT_BY.NAME_DESCENDING, 'Name (Z to A) ↓'],
-            [SORT_BY.LATEST_UPDATE, 'Newest (Updated) ↓'],
-            [SORT_BY.OLDEST_UPDATE, 'Oldest (Updated) ↑']
+            [SORT_BY.DATE_DESCENDING, {category: 'Newest (Created)', selected: 'Created ↓'}],
```

Comment:
can we replace 
category: 'Newest (Created)'
with
category: 'Created - Newest'